### PR TITLE
Really hide unused items. And fix the sticky sections!

### LINKED ISSUE #85: Sticky headers are broken
Because of https://github.com/gaia-components/fast-list/blob/de07ebcaa0ca7334769ec3aae3ff302b68e0c34a/fast-list.js#L45

We need to fix it without breaking variable height support.

I'll get to it, but probably not today.

### PR DESCRIPTION
**Pull request by etiennesegonzac**
Closes #85

### CODE REVIEW

File: test/fast-list-test.js [hunk 1]
```diff
@@ -136,22 +135,22 @@ suite('FastList >', function() {
...
-      });
 
-      suite('then grows back', function() {
-        setup(function() {
-          var bigData = createDummyData(1000);
-          source.data = bigData;
-          fastList.reloadData();
-          scheduler.mutation.yield();
-        });
+        suite('then grows back', function() {
```
Comment (etiennesegonzac) [inline test/fast-list-test.js:139]: my style of nested unit tests usually serves me well, but I missed a nesting level here before :)

File: fast-list.js [hunk 2]
```diff
@@ -304,6 +303,9 @@ FastList.prototype = {
         item.dataset.detailPopulated = false;
         source.populateItem(item, i);
         item.dataset.populated = true;
+        if (item.style.display === 'none') {
+          item.style.removeProperty('display');
```
Comment (wilsonpage) [inline fast-list.js:307]: Didn't know about this method! Any advantage to using `item.style.display = ''`?

**Comment by wilsonpage** [comment]
One lint error: `test/fast-list-test.js: line 127, col 13, Missing semicolon.`

then ... r+!

**Comment by wilsonpage** [comment]
This is a much more efficient solution, nice one :)

**Comment by wilsonpage** [comment]
NIT: Just spotted you need to update the example `<title>` in `examples/variable/index.html`.

File: fast-list.js (hunk 2 above)
Comment (etiennesegonzac) [inline fast-list.js:307]: not that I know off, I think I picked it up from @mcav :)

**Pull request merged**